src/main/content/_assets/js/docs.js

```diff
@@ -46,13 +46,23 @@ function areLinksVisible(links){
 
 function selectDocInToc(){
     let currentHref = window.location.href;
-    let selectedFile = '/docs/ref/general' + currentHref.substring(currentHref.lastIndexOf('/'));
-    if(selectedFile !== '/docs/ref/general/docs-welcome.html'){
+    let pathName = location.pathname.substring(0, 11);
+    let selectedFile = `${pathName}/ref/general` + currentHref.substring(currentHref.lastIndexOf('/'));
+    if(selectedFile !== `${pathName}/ref/general/docs-welcome.html`){
         $(`a[href$="${selectedFile}"]`).addClass('active-doc')
         $( `a[href$="${selectedFile}"]` ).parent().parent().parent().find('.toc-category').click();
     }
 }
 
+function initialDocsRedirect() {
+    let docVersions = JSON.parse($('#docs-version-json').html() || '{}');
+    let currentPath = window.location.pathname;
+    if (currentPath === '/docs/' && docVersions){
+        window.location.pathname = `/docs/${docVersions[0]}`;
```

Comment:
add comment here about if no doc version is specified in the URL we will redirect to the first version (which is the latest version because its sorted, right?)